analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -26,10 +26,63 @@ histologies_file <- file.path("..", "..", "data", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+```{r}
+# Filter to WXS tumor with no germline sex estimate
+no_estimate <- histologies_df %>% 
+  filter(sample_type == "Tumor") %>% 
+  filter(experimental_strategy == "WXS") %>%  
+  filter(is.na(germline_sex_estimate)) %>%
+  select(-germline_sex_estimate)
+
+# Extract participant ID with no germline sex estimate
+no_estimate_participants <- no_estimate %>% 
+  pull(cohort_participant_id) %>% 
+  unique()
+
+# Find out matching WGS - currently this has no entry for v9
+no_estimate_participants_match_wgs <- histologies_df %>% 
+  filter(cohort_participant_id %in% 
+  no_estimate_participants) %>% 
+  filter(experimental_strategy == "WGS") %>% 
+  filter(sample_type == "Tumor") %>% 
+  select(cohort_participant_id, germline_sex_estimate)
```

Comment:
```suggestion
no_estimate_participants_match_wgs <- histologies_df %>% 
  filter(Kids_First_Participant_ID %in% no_estimate_participants & experimental_strategy == "WGS" & sample_type == "Tumor") %>% 
  select(Kids_First_Participant_ID, germline_sex_estimate)
```

Condensing to use `filter()` only once and add `Kids_First_Participant_ID` here. This actually gives 10 participants that match now.

Reply:
Done